probe/core/src/main/client/app.js

```diff
@@ -520,6 +571,32 @@ Probe.InvocationListController = Ember.ObjectController.extend({
 
 Probe.InvocationDetailController = Ember.ObjectController.extend({});
 
+Probe.EventsController = Ember.ObjectController.extend({
+    init : function() {
+        this._super();
+        this.set('initialized', true);
+        this.set('eventKinds', eventKinds);
+    },
+    eventInfo : '',
+    type : '',
+    qualifiers : '',
+    page : 1,
+    kind : '',
```

Comment:
All values referenced from a select view must have the "empty/ANY" value set to `null` otherwise ember.js sends some odd additional requests. I'm not sure why. This applies to `clearFilters` as well.